_pages/01-applicants.md

```diff
@@ -1,23 +1,25 @@
 ---
 layout: applicant
 title: Information for Applicants
-description: <b>Due to COVID-19, there will be no in-person event in 2020. 
-    A virtual event around August is being discussed.
-    Applications will open soon.
-    Please check back in early May.</b><br><br>
+description: <b>Due to COVID-19, OceanHackWeek (OHW) 2020 will take place as a virtual, online event.
```

Comment:
What was the verdict OceanHackWeek or OceanHackweek?

Reply:
OceanHackWeek (it's what settled on for the NSF proposal)